scapy/layers/l2.py

```diff
@@ -506,7 +506,10 @@ def mysummary(self):
 
 def l2_register_l3_arp(l2, l3):
     # type: (Type[Packet], Type[Packet]) -> Optional[str]
-    return getmacbyip(l3.pdst)
+    # TODO: support IPv6?
```

Comment:
What is the goal of this change?
FTR, we do not need to support IPv6 here.

Reply:
The comment or the changes to the function?
`ARP`'s pdst can be an IPv6 address in which case calling `getmacbyip` on it wouldn't make sense. I was unsure what we should do in that case

Reply:
I did not know that ARP could carry IPv6 address. In that case, I might be useful to display a warning for everytheing else than IPv4 explaining that the address type is not supported.

Reply:
Good idea. I've done that.
This is ready for review !